packages/neos-ui/src/Sagas/UI/ContentCanvas/index.js

```diff
@@ -15,7 +16,16 @@ function * watchNodeCreated() {
         yield put(actions.UI.ContentCanvas.setSrc($get('uri', node)));
     });
 }
+/**
+ * Load newly created page into canvas
+ */
+function * watchCanvasUpdateToChangeTitle() {
+    yield * takeLatest(actionTypes.UI.ContentCanvas.STOP_LOADING, () => {
+        document.title = iframeDocument().title;
```

Comment:
Sure! Any suggestions? Otherwise I would merge it for now and we start a discussion in a ticket about a prefix